lib/rucio/core/message.py

```diff
@@ -64,33 +66,44 @@ def add_message(event_type, payload, session=None):
         try:
             services_list = config_get('hermes', 'services_list')
         except (NoOptionError, NoSectionError, RuntimeError):
-            services_list = None
+            services_list = 'influx,activemq,elastic,email'
```

Comment:
No caching needed around common.config_get, it's already cached within.
Wouldn't it be better to make the default only `activemq,email` - to mirror hermes1 behaviour?